packages/grafana-ui/src/components/InfoBox/InfoBox.mdx

```diff
@@ -0,0 +1,24 @@
+import { Meta, Props } from '@storybook/addon-docs/blocks';
+import { InfoBox } from './InfoBox';
+
+<Meta title="MDX|InfoBox" component={InfoBox} />
+
+# InfoBox
+A component for displaying information in different levels of severity (Success, Warning, Error, Info).
+
+
+### Example usage
+This component can be found in the settings for various data sources.
+
+
+```tsx
+<InfoBox
+  title="Example info box"
+  severity='info'
+  url="http://url.to.some.documentation"
+  onDismiss={() => dissmisInfoBox}
```

Comment:
Shouldn't this be either `onDismiss={() => dissmisInfoBox()}` or `onDismiss={dissmisInfoBox}`?